materials/dplyr.md

```diff
@@ -110,3 +110,23 @@ query <- "SELECT year, month, day, genus, species
           ON surveys.species_id = species.species_id"
 tbl(portaldb, sql(query))
 ```
+
+* Queries and data manipulation functions return similar results with various headings.
+
```

Comment:
Students should understand the various ways that tables can be derived and stored (external vs. local) using the `dplyr` database interface. One construct using `tbl( sql(query))` is SQL flavored and the other is `dplyr` verbs that manipulate a database table. They should be able to identify the different table output from the external database or a local data frame.